test/ng/compileSpec.js

```diff
@@ -5629,6 +5629,34 @@ describe('$compile', function() {
       });
 
 
+      //see issue https://github.com/angular/angular.js/issues/12936
+      it('should use the proper scope when being the root element of a replaced directive', function() {
+        module(function() {
+          directive('parent', valueFn({
+            scope: {},
+            replace: true,
+            template: '<div trans>{{x}}</div>',
+            link: function(scope, element, attr, ctrl) {
+              scope.x = 'iso';
+            }
+          }));
+          directive('trans', valueFn({
+            transclude: 'content',
```

Comment:
@lgalfaso : Originally the possible values for `transclude` were indeed `"content"` and `"element"` but it got shorthanded to `true` and `"element"`. So this is a valid value.